woocommerce/woocommerce
# CYS - Hide theme and default palettes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40822 

This PR hides theme and default palettes on the assember hub page.

Note: I've played with settings, but I could not find a way to hide them on Woo Express. Since theme and default only show up on Woo Express, I've decided to use CSS to hide them.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Follow pdibGW-2pG-p2 and create a new Woo Express site for testing 
2. Start `Customize your store` task and proceed to the assember hub page.
3. Click on `Change the color palette`
4. Click on `OR CREATE YOUR OWN`
5. Click on the one of the options and confirm theme and default palette are not rendered.



### Changelog entry




<details>

#### Significance



-   [] Minor

#### Type




#### Message 

CYS - Hide theme and default palettes on the assember hub page.

#### Comment 

</details>
